packages/buckets/src/api/index.ts

```diff
@@ -405,11 +423,15 @@ export async function bucketsPushPath(
   input: any,
   opts?: PushOptions,
   ctx?: ContextInterface,
-) {
+): Promise<PushPathResult> {
```

Comment:
We have been lax with these. Our docs building will be happy with use for this. I've added this as a linter check, so it should be easier to keep this up to date.